src/Common/tests/System/Net/Http/LoopbackServer.cs

```diff
@@ -128,6 +131,75 @@ public static Task CreateClientAndServerAsync(Func<Uri, Task> clientFunc, Func<L
             return lines;
         }
 
+        public async Task<List<string>> AcceptConnectionPerformAuthenticationAndCloseAsync(string authenticateHeaders)
```

Comment:
Can we move this code, and the related code like Is*AuthTokenValid, somewhere other than LoopbackServer?  Maybe AuthTestHelpers.cs or something like that?

I'd like to keep LoopbackServer itself minimal.

Reply:
Sure